### PR TITLE
Change token from 3mcloudlibrary to cloudlibrary

### PR DESCRIPTION
**Pull request by Saklad5**
_If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so._

After making all changes to the cask:
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:
- [ ] Named the cask according to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [ ] Checked that the cask was not already refused in [closed issues](https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed).

Add cloudLibrary.app v2.0.1610051750
Fix macOS dependencies

### CODE REVIEW

**Comment by Saklad5** [comment]
Why aren't we running build tests on macOS Sierra yet?

**Comment by reitermarkus** [comment]
> Why aren't we running build tests on macOS Sierra yet?

Travis doesn't have a Sierra image yet.

File: Casks/cloudlibrary.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    version '2.0.1610051750'
+    sha256 'f867a2858ca339555d655adec2e1e9f2e682381b8bb8745006f9d35cb87aa480'
+
+    url "http://download.yourcloudlibrary.com/apps/mac/cloudLibrary-#{version}.pkg"
+    pkg "cloudLibrary-#{version}.pkg"
+
+    uninstall pkgutil: 'com.yourcloudlibrary.mac'
+  end
+
+  appcast 'http://www.yourcloudlibrary.com/index.php/en-us/downloads/software?format=rss',
```
Comment (vitorgalvao) [inline Casks/cloudlibrary.rb]: Move `appcast` inside the `else`.

File: Casks/cloudlibrary.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+          checkpoint: '948716fbbe4a3af107c7e73424608615bf53623b0110dc24fa456a2ce06dad92'
+  name '3M Cloud Library'
+  name 'cloudLibrary'
```
Comment (vitorgalvao) [inline Casks/cloudlibrary.rb]: Remove this name, as it adds nothing.

File: Casks/cloudlibrary.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+cask 'cloudlibrary' do
+  if MacOS.version <= :lion
+    version '1.50.735564'
+    sha256 '569b78d4f78054518b7ea87678637949408ca7cd1fd90baf2221d92fff35cdfc'
+
+    url "http://www.yourcloudlibrary.com/index.php/en-us/downloads/software/2-3mcloudlibrary-live-version-#{version.dots_to_hyphens}/file"
+    app '3MCloudLibrary.app'
```
Comment (vitorgalvao) [inline Casks/cloudlibrary.rb]: Put an empty line above this one.

File: Casks/cloudlibrary.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  else
+    version '2.0.1610051750'
+    sha256 'f867a2858ca339555d655adec2e1e9f2e682381b8bb8745006f9d35cb87aa480'
+
+    url "http://download.yourcloudlibrary.com/apps/mac/cloudLibrary-#{version}.pkg"
+    pkg "cloudLibrary-#{version}.pkg"
```
Comment (vitorgalvao) [inline Casks/cloudlibrary.rb]: And another above this one.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Made the requested changed myself, since your branch allowed for it.

**Review by vitorgalvao — APPROVED** [review]

**Comment by Saklad5** [comment]
The appcast applies to both the old version and the new version. And the style check arranged the stanzas like that, not me.

**Comment by Saklad5** [comment]
Is it really that hard to wait a few hours for a reply?

**Comment by Saklad5** [comment]
Furthermore, even the source website is inconsistent about whether the name is "3MCloudLibrary" or "cloudLibrary". It took me ages to confirm that the former had been changed to the latter officially. Anyone looking for this Cask might use either name.

**Comment by jawshooah** [comment]
> And the style check arranged the stanzas like that, not me.

The style check isn't smart enough yet to rearrange stanzas inside `if...else` blocks, sorry about that.

> Is it really that hard to wait a few hours for a reply?

Let's keep the tone civil, eh? We deal with a pretty high volume of PRs, so fixing and closing them out takes priority over waiting for a user to implement requested changes if a maintainer has time to do it themselves.

**Comment by Saklad5** [comment]
Sorry, I just was annoyed that he changed information about the cask without waiting a few minutes for a reply. If it was just syntax that he changed, I wouldn’t be so irritated.

**Comment by reitermarkus** [comment]
> he changed information about the cask

Is the information as it is now correct?

**Comment by Saklad5** [comment]
The appcast stanza applies to both versions, not just the newly named one.

On Oct 15, 2016, at 10:54 AM, Markus Reiter <notifications@github.com> wrote:

he changed information about the cask

Is the information as it is now correct?

**Comment by vitorgalvao** [comment]
> The appcast stanza applies to both versions, not just the newly named one.

From the point there is an older version of the app for an older version of the OS, the appcast starts to apply only to the newest one, according to our system. Unless there’s a version for Lion and earlier being developed at the same time as this new one, which does not seem to be the case at all, it only applies to the newest case. Simply because the older version is still present in the appcast, it does not mean it still applies. It’s present as an old version and nothing more.

In addition, I made the changes to save you work, I don’t really see why you chose to take that as an insult. My comments served to inform you of the changes.

Thank you for the contribution.

**Comment by Saklad5** [comment]
Who said I took it as an insult?

On 15 October 2016, at 8:48 PM, Vítor Galvão <notifications@github.com> wrote:

—

**Pull request merged**